AdblockPlusSafariTests/AdblockPlusSafariTests.m

```diff
@@ -1,6 +1,6 @@
 /*
  * This file is part of Adblock Plus <https://adblockplus.org/>,
- * Copyright (C) 2006-2016 Eyeo GmbH
+ * Copyright (C) 2006-2015 Eyeo GmbH
```

Comment:
Nit: Why change it from 2016 to 2015 in 2017? :smile: Let's not change it at all to not have any unrelated changes in here. We should update the existing license headers in a separate commit.